bin/upgrading/proplists.dat

```diff
@@ -62,14 +62,6 @@ userproplist.city:
   multihomed: 0
```

Comment:
Should be no actual changes to this file, but I re-dumped it so everything is ordered now instead of just manually updated lol.